FGEGraphics/GraphicsHelpers/Models/ShapeGenerators.cs

```diff
@@ -0,0 +1,408 @@
+﻿//
+// This file is part of the Frenetic Game Engine, created by Frenetic LLC.
+// This code is Copyright (C) Frenetic LLC under the terms of a strict license.
+// See README.md or LICENSE.txt in the FreneticGameEngine source root for the contents of the license.
+// If neither of these are available, assume that neither you nor anyone other than the copyright holder
+// hold any right or permission to use this software until such time as the official license is identified.
+//
+
+using FGECore.MathHelpers;
+using FGECore.ModelSystems;
+using System;
+using System.Collections.Generic;
+using OpenTK.Mathematics;
+using FGECore.CoreSystems;
+
+namespace FGEGraphics.GraphicsHelpers.Models;
+
+/// <summary>Generates shapes and returns them as a model.</summary>
+public static class ShapeGenerators
+{
+    /// <summary>
+    /// Generates a 3D sphere model.
+    /// Note that the stacks should be lower than the slices to avoid incorrect normals a good example is 10 stacks and 60 slices.
+    /// </summary>
+    /// <param name="radius"></param>
```

Comment:
?

Reply:
If the stacks are higher than the slices you get this:

![bandicam 2024-03-04 09-33-33-052](https://github.com/FreneticLLC/FreneticGameEngine/assets/97306922/6e2787f9-55f3-4d14-8cd8-89f693078b01)

Similar situation with torus will add warning for sphere.

Reply:
the question mark was re the empty documentation entries